general/payment_acquirers/stripe.rst

```diff
@@ -0,0 +1,27 @@
+======
+Stripe
+======
+
+`Stripe <https://stripe.com/>`__ is a United States-based online payment solution
+provider, allowing businesses to accept **credit cards** and other payment methods.
+
+Local Payment Methods
+=====================
+
+`Local payment methods <https://stripe.com/payments/payment-methods-guide>`__ are payment methods which are
+only available for certain merchants and customers countries and currencies.
+
+Odoo supports the integration of the following local payment methods:
+
+- Bancontact
+- EPS
+- Giropay
+- iDeal:
+- Przelewy24 (P24)
+
+In order to support one local payment method, you must add its icon to the supported payment method icons:
+Payment Acquirers > Stripe > Configuration > Supported Payment Icons.
```

Comment:
```suggestion
To offer a given local payment method to your customers, it must be listed as a supported payment icon in Stripe. Go to :menuselection:`Payment Acquirers --> Stripe --> Configuration` and add the desired payment method in **Supported Payment Icons**.
```
@jcs-odoo Could you please check this writing?

@AdrienHorgnies 
- https://www.odoo.com/documentation/user/14.0/contributing/documentation/rst_guidelines.html#use-the-menuselection-directive
- This could benefit from an illustration (don't forget to use *pngquant* to compress it before committing).

Reply:
done

Reply:
sure, I post my suggestion as a different review. :)